build/windows/installer/conf/fluent.conf

```diff
@@ -46,6 +61,23 @@
   </record>
 </filter>
 
+<match mdm.cadvisorperf**>
+  @type mdm
+  @log_level debug
+  <buffer>
+    @type file
+    path /etc/omsagentwindows/out_mdm_cdvisorperf.buffer
+    overflow_action drop_oldest_chunk
+    chunk_limit_size 4m
```

Comment:
I think, we should add queue_limit_length 20 similar to all other plugins. This will help us to buffer upto 80MB in case of transient failures.

Reply:
I have reused the configuration we have in Linux. Is there any reason why this was not in Linux and should be included in Windows?